app/views/application/_analytics.html.erb

```diff
@@ -1,7 +0,0 @@
-<% if ENV["SEGMENT_KEY"] %>
```

Comment:
this was never used. I know that because 18F does not have an ATO for Segment :)